parsec/sequester_export_reader.py

```diff
@@ -52,6 +52,90 @@ class InconsistentWorkspaceError(RealmExportReaderError):
     pass
 
 
+# This is (sligthly patched) copy of `parsec.core.fs.path.FsPath`, the idea is
+# we cannot import stuff from `parsec.core` if only backend dependencies are
+# installed.
+# On top of that it's complicated to move `FsPath` out of `parsec.core` given
+# it purposefuly uses `parsec.core.fs.exceptions.FSNameTooLongError` exception
+# (that's the part we have patched out here).
+class FsPath:
```

Comment:
This implementation could be replaced  by `pathlib.PurePath` since all we need is a path representation without IO.

This would avoid the code duplication where most methods are not used anyway.

Reply:
That's a great idea ! I've updated my PR with it ;-)